schema/deploy/roles/power-bi-exporter/create.sql

```diff
@@ -0,0 +1,10 @@
+-- Deploy seattleflu/id3c-customizations:roles/power-bi-exporter/create to pg
+
+begin;
+
+create role "power-bi-exporter";
+
+comment on role "power-bi-exporter" is
+    'For exporting data to Power BI';
```

Comment:
This name seems overly generic to me since Power BI is just a tool, an implementation detail. What about something more specific, like `scan-dashboard-exporter`, that would still make sense even if, for example, the dashboard switches from Power BI to Tableau?